composer/datasets/in_context_learning_evaluation.py

```diff
@@ -293,6 +293,13 @@ def __init__(
         hf_parsing_map: Optional[Dict] = None,
         generation_kwargs: Optional[Dict] = None,
     ):
+        warnings.warn(
```

Comment:
you can just put this on the base class right? all instantiation paths end up there